src/main/resources/orgs.csv

```diff
@@ -0,0 +1,5 @@
+id,name,parentId,commitOptions
```

Comment:
In the design, we agreed to make this part of repos.csv as extra columns, not a separate csv.

Reply:
after discussion we decided to not include commitOptions. Dropping this file